README.md

```diff
@@ -50,6 +50,9 @@ Source the environment and provide the password:
 $ source .env
 ```
 
+### OAuth2 Authentication
+The SQL-shell uses the [visier-connector](/visier/connector-python) to connect to Visier. Please reference this connector for details how to set up OAuth 2.0 with Visier.
```

Comment:
The SQL-shell uses the [visier-connector](/visier/connector-python) to connect to Visier. For more information about setting up OAuth 2.0 with Visier, see [visier-connector](/visier/connector-python).